src/test/groovy/org/shipkit/internal/gradle/util/StringUtilTest.groovy

```diff
@@ -0,0 +1,14 @@
+package org.shipkit.internal.gradle.util
+
+import spock.lang.Specification
+
+class StringUtilTest extends Specification {
+
+    def "knows if empty"() {
+        expect:
+        StringUtil.isEmpty(null)
+        StringUtil.isEmpty("")
+        !StringUtil.isEmpty(" ")
+        !StringUtil.isEmpty("xx")
```

Comment:
Not very much in the [Spock style](http://spockframework.org/spock/docs/1.1/data_driven_testing.html) (where parameterized tests are easy to write).